Truck.js

```diff
@@ -1,6 +1,40 @@
 var Car = require('./Car.js');
 
-//Implement your Truck here
-function Truck() {};
+//Implement your Motorcycle here
+function Truck(make, model, year, color, seats, passengers, capacity) {
+	Car.call(this, make, model, year, color, seats, passengers);
+	this.capacity = capacity || 0;
+	this.cargo = 0;
+};
+
+Truck.prototype = Object.create(Car.prototype);
+Truck.prototype.constructor = Truck;
+
+var a = new Car("blue");
+var s = new Truck("a", "b", "c", "d", "e", "f", "g");
+
+s instanceof Car;
+a instanceof Truck;
+
+// Truck.prototype.capacity = function(capacity){
+// 	this.capacity = capacity || 0;
+// };
+
+
+Truck.prototype.load = function(pounds) {
+	if (pounds <= this.capacity) {
+		this.cargo += pounds;
+	return true ;
+	} return false;
+};
+
+Truck.prototype.unload = function(pounds) {
+	if (pounds <= this.cargo) {
+		this.cargo -= pounds;
+		return true;
+	} return false;
+};
+
```

Comment:
With the above two methods, we need to make sure we're accounting for the cargo already loaded into the truck, and that it is below the maximum capacity:
```
Truck.prototype.load = function(amount) {
	if (amount + this.cargo <= this.capacity) {
		this.cargo += amount;
		return true;
	}
	return false;
}

Truck.prototype.unload = function(amount) {
	if (amount <= this.cargo) {
		this.cargo -= amount;
		return true;
	}
	return false;
}
```